src/components/LightningWallet.vue

```diff
@@ -842,7 +844,26 @@ export default {
       this.send.isSending = false;
     },
     async createInvoice() {
+      // Check that there is enough in the recieve
+      if (!this.maxRecieve || this.recieve.amount > this.maxRecieve) {
```

Comment:
Why do you think we have to check for the existence of `maxRecieve`? Even if it's 0, `this.recieve.amount > this.maxRecieve` will do the job.

Reply:
I think I was seeing some cases where it was undefined, which doesn't really make sense since it's set to 0 in the store. I'll confirm this.